2024-05-16 19:23:13,429:INFO:Running command: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"1"' ++distinguisher.attack_id_1='"2_1"' ++distinguisher.output_2='"2"' ++distinguisher.attack_id_2='"1_1"' 
2024-05-16 19:23:13,430:INFO:Saving results to ./results/stationary_distribution/robustness_analysis/entropy_4/distinguisher_results/1_2_1-2_1_1_05_16_llama_mass_huge_batch.log
2024-05-16 19:24:41,808:ERROR:Command failed with error: None
2024-05-16 19:24:41,808:INFO:Running command: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"1"' ++distinguisher.attack_id_1='"3_1"' ++distinguisher.output_2='"2"' ++distinguisher.attack_id_2='"1_1"' 
2024-05-16 19:24:41,808:INFO:Saving results to ./results/stationary_distribution/robustness_analysis/entropy_4/distinguisher_results/1_3_1-2_1_1_05_16_llama_mass_huge_batch.log
2024-05-16 19:26:16,216:ERROR:Command failed with error: None
2024-05-16 19:26:16,216:INFO:Running command: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"1"' ++distinguisher.attack_id_1='"2_1"' ++distinguisher.output_2='"3"' ++distinguisher.attack_id_2='"2"' 
2024-05-16 19:26:16,216:INFO:Saving results to ./results/stationary_distribution/robustness_analysis/entropy_4/distinguisher_results/1_2_1-3_2_05_16_llama_mass_huge_batch.log
2024-05-16 19:27:51,047:ERROR:Command failed with error: None
2024-05-16 19:27:51,048:INFO:Running command: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"1"' ++distinguisher.attack_id_1='"2_1"' ++distinguisher.output_2='"3"' ++distinguisher.attack_id_2='"4"' 
2024-05-16 19:27:51,048:INFO:Saving results to ./results/stationary_distribution/robustness_analysis/entropy_4/distinguisher_results/1_2_1-3_4_05_16_llama_mass_huge_batch.log
2024-05-16 19:29:16,770:ERROR:Command failed with error: None
2024-05-16 19:29:16,771:INFO:Running command: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"1"' ++distinguisher.attack_id_1='"3_1"' ++distinguisher.output_2='"3"' ++distinguisher.attack_id_2='"2"' 
2024-05-16 19:29:16,771:INFO:Saving results to ./results/stationary_distribution/robustness_analysis/entropy_4/distinguisher_results/1_3_1-3_2_05_16_llama_mass_huge_batch.log
2024-05-16 19:30:51,959:ERROR:Command failed with error: None
2024-05-16 19:30:51,959:INFO:Running command: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"1"' ++distinguisher.attack_id_1='"3_1"' ++distinguisher.output_2='"3"' ++distinguisher.attack_id_2='"4"' 
2024-05-16 19:30:51,959:INFO:Saving results to ./results/stationary_distribution/robustness_analysis/entropy_4/distinguisher_results/1_3_1-3_4_05_16_llama_mass_huge_batch.log
2024-05-16 19:32:26,518:ERROR:Command failed with error: None
2024-05-16 19:32:26,518:INFO:Running command: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"2"' ++distinguisher.attack_id_1='"1_1"' ++distinguisher.output_2='"3"' ++distinguisher.attack_id_2='"2"' 
2024-05-16 19:32:26,518:INFO:Saving results to ./results/stationary_distribution/robustness_analysis/entropy_4/distinguisher_results/2_1_1-3_2_05_16_llama_mass_huge_batch.log
2024-05-16 19:34:01,991:ERROR:Command failed with error: None
2024-05-16 19:34:01,991:INFO:Running command: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"2"' ++distinguisher.attack_id_1='"1_1"' ++distinguisher.output_2='"3"' ++distinguisher.attack_id_2='"4"' 
2024-05-16 19:34:01,991:INFO:Saving results to ./results/stationary_distribution/robustness_analysis/entropy_4/distinguisher_results/2_1_1-3_4_05_16_llama_mass_huge_batch.log
2024-05-16 19:35:25,735:ERROR:Command failed with error: None
2024-05-16 19:35:25,735:INFO:[(None, None), (None, None), (None, None), (None, None), (None, None), (None, None), (None, None), (None, None)]
Command succeeded: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"1"' ++distinguisher.attack_id_1='"2_1"' ++distinguisher.output_2='"2"' ++distinguisher.attack_id_2='"1_1"' 
Output:
None
Command succeeded: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"1"' ++distinguisher.attack_id_1='"3_1"' ++distinguisher.output_2='"2"' ++distinguisher.attack_id_2='"1_1"' 
Output:
None
Command succeeded: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"1"' ++distinguisher.attack_id_1='"2_1"' ++distinguisher.output_2='"3"' ++distinguisher.attack_id_2='"2"' 
Output:
None
Command succeeded: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"1"' ++distinguisher.attack_id_1='"2_1"' ++distinguisher.output_2='"3"' ++distinguisher.attack_id_2='"4"' 
Output:
None
Command succeeded: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"1"' ++distinguisher.attack_id_1='"3_1"' ++distinguisher.output_2='"3"' ++distinguisher.attack_id_2='"2"' 
Output:
None
Command succeeded: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"1"' ++distinguisher.attack_id_1='"3_1"' ++distinguisher.output_2='"3"' ++distinguisher.attack_id_2='"4"' 
Output:
None
Command succeeded: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"2"' ++distinguisher.attack_id_1='"1_1"' ++distinguisher.output_2='"3"' ++distinguisher.attack_id_2='"2"' 
Output:
None
Command succeeded: python distinguisher.py +distinguisher=llama3 ++distinguisher.log_suffix=05_16_llama_mass_huge_batch ++distinguisher.entropy='"4"' ++distinguisher.output_1='"2"' ++distinguisher.attack_id_1='"1_1"' ++distinguisher.output_2='"3"' ++distinguisher.attack_id_2='"4"' 
Output:
None
